Products/CMFPlone/skins/plone_form_scripts/discussion_reply.cpy

```diff
@@ -14,6 +14,9 @@ from Products.CMFPlone import PloneMessageFactory as _
 mtool = getToolByName(context, 'portal_membership')
 dtool = getToolByName(context, 'portal_discussion')
 req = context.REQUEST
+if req.get('REQUEST_METHOD', '').upper() != 'POST':
+    from zExceptions import Forbidden
```

Comment:
In Python skin scripts I would say: absolutely.  In such a script, every line that gets executed, is checked for security, which takes relatively long. So you want to avoid an import unless you really need it.